ikenna-portfolio/portfolio/src/main/webapp/index.html

```diff
@@ -47,97 +47,146 @@ <h3>I'm a Googler!!</h3>
+          <option value="cyclegan">CycleGAN</option>
+          <option value="deep-photo">Deep Photo Style Transfer</option>
+          <option value="msg-sys">Message System</option>
+        </select>
+      </form><hr><br>
+      <div id="projects-pics" class="row"></div>
       <br>
       <div class="row">
         <div class="col-sm-8">
-          <p id="pro-desc" class="desc"> </p>
+          <p id="pro-desc" class="desc"></p>
         </div>
         <div class="col-sm-4">
-          <p id="pro-links" class="desc"> </p>
+          <p id="pro-links" class="desc"></p>
         </div>
       </div>
     </div>
 
-    <!-- Fourth Container-->
-    <div class="container-fluid bg-4 text-center">    
-      <h3 class="margin">Calisthenics Goals</h3><br>
+    <!-- Calisthenics Container-->
+    <div id="calisthenics-sect" class="container-fluid bg-4 text-center">
+      <h3 class="margin">Calisthenics Goals</h3>      
+      <form action="/calisthenics" method="POST">
+        <button class="btn btn-info">Go!</button>
+        <select name="cal" id="cal" class="btn btn-secondary dropdown-toggle">
+          <option value="">--Please choose an exercise--</option>  
+          <option value="vs">V Sit</option>
+          <option value="bl">Back Lever</option>
+          <option value="ps">Pistol Squat</option>
+          <option value="fl">Front Lever</option>
+          <option value="hspu">Freestanding Handstand Push Up</option>
+        </select>
+      </form><br>
       <div class="row">
-        <div class="column">
-          <button onclick="changeVSit()">V Sit</button>
-        </div>
-        <div class="column">
-          <button onclick="changeBackLever()">Back Lever</button>
-        </div>
-        <div class="column">
-          <button onclick="changePistolSquat()">Pistol Squat</button>
-        </div>
-        <div class="column">
-          <button onclick="changeFrontLever()">Front Lever</button>
-        </div>
-        <div class="column">
-          <button onclick="changeHSPU()">Freestanding Handstand Push Up</button>
-        </div>
-      </div>
-      <br>
-      <div class="row">
-        <div class="column">
+        <div class="col-sm-6">
           <img src="" id="cal-image1" class="img-responsive margin image-format" alt="">
           <h3 id="goal"></h3>
         </div>
-        <div class="column"> 
+        <div class="col-sm-6"> 
           <img src="" id="cal-image2" class="img-responsive margin image-format" alt="">
           <h3 id="current"></h3>
         </div>
       </div>
     </div>
 
+    <!--Comments Container-->
+    <div id="comments-sect" class="container-fluid bg-5 text-center">
+      <h3 class="margin">Any Questions, Comments, and Concerns? Post Them Below!</h3>
+      <h4 class="margin">We promise to take your posts seriously >:)</h4>
+      <form action="/data" method="POST" class="desc">
+        <div class="comment-wrapper">
```

Comment:
Is it possible to combine comment-wrapper, panel-info and panel-body to make simpler? You can have multiple classes with a single html element?

Reply:
done